packages/features/bookings/lib/handleNewBooking.ts

```diff
@@ -1626,6 +1631,7 @@ async function handler(
     return resultBooking;
   };
   // For seats, if the booking already exists then we want to add the new attendee to the existing booking
+  // This also means if booker doesn't sends bookingUid it will create a separate booking
```

Comment:
I might be wrong on this but looks like new booker or handleNewBooking doesn't check if there is already a booking created with same start end date? I saw a bug on that recently.
@joeauyeung @zomars

Reply:
It does after the date range overhaul; a busy time would be excluded within the date ranges so would prevent the booking.

Reply:
Awesome @emrysal, will remove comment now
```suggestion
```